web2/htdocs/index.html

```diff
@@ -243,6 +243,36 @@ <h2>CLI Authentication Credentials</h2>
             </div>
           </div>
     <!-- }}} --></script>
+    <script type="text/html" id="tpl--disasterkey"><!-- {{{ -->
+        [[#
+          {disasterkey} template
+  
+          Displays the Shield Disaster Key, so that operators can restore
+          fixed-key backups in the event of a disaster.
+        ]]
+            <div id="cliauth" class="frontdoor">
+              <div>
+                <h1>SHIELD<span>Production<span></h1>
+                <div class="dialog">
+                  <h2>Shield Disaster Key</h2>
+                  <img src="/i/terminal.png" class="highlight">
+  
+                  <p>Here is your SHIELD Disaster Key:</p>
+  
+                  <pre id="disasterkey">[[= _.disaster_key ]]</pre>
+  
+                  <p id="dr-explain">
+                    You will need to save this key in a secure location
+                    for use in recovery of fixed-key backups in a disaster scenario.
+                    Fixed-key backups must be used to backup shield itself. Once you
+                    leave this screen, there is no way to access this key or corresponding
+                    fixed-key backups unless you have saved it!
```

Comment:
We should either change all the other "Disaster Recovery" references to "Fixed Key", or change these "Fixed Key" references to "Disaster Recovery", for consistency.